pkgs/development/python-modules/pytest-flake8/default.nix

```diff
@@ -0,0 +1,25 @@
+{lib, buildPythonPackage, fetchPypi, pytest, flake8}:
+
+buildPythonPackage rec {
+  name = "${pname}-${version}";
+  pname = "pytest-flake8";
+  version = "0.8.1";
+
+  propagatedBuildInputs = [ pytest flake8 ];
```

Comment:
Ok, thanks. I did that fix now. Is it ok?

In general, I would love to see that nix wouldn't replace Python packages with their new versions but would contain all versions that have ever been added to nix. One of them would a default and built by Hydra. Perhaps a few others versions too in some cases (like pytest). But then users could install different versions of packages similarly as with pip. Each Python package could have a list of sources and adding a new version would just mean adding a new version option, url and sha256. Also, using a different version would make the package point to that requested version, so there wouldn't be any problem defining `pytest` as a `propagatedBuildInputs` because `pytest` points to the version the user wants to install. Anyway, that's a different topic and a more general comment, not sure it would work with nix though.